ui/app/mirrors/edit/[mirrorId]/cdc.tsx

```diff
@@ -284,12 +289,18 @@ export function CDCMirror({
 
   useEffect(() => {
     if (typeof window !== 'undefined') {
-      setSelectedTab(localStorage?.getItem('mirrortab') || 'tab1');
+      setSelectedTab(
+        parseInt(localStorage?.getItem(LocalStorageTabKey) || '0')
```

Comment:
```suggestion
        parseInt(localStorage?.getItem(LocalStorageTabKey)) | 0
```
This'll avoid being NaN if localStorage has some non-integer truthy string